frontend/src/presentation/components/LandingComponent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Card } from "antd";
+import styles from "./LandingComponent.module.css";
+import { AddQuestionButton } from "./buttons/AddQuestionButton";
+
+interface LandingComponentProps {
+  onAddQuestion: () => void;
+}
+
+export const LandingComponent: React.FC<LandingComponentProps> = ({ onAddQuestion }) => (
+  <div className={styles.cardWrapper}>
+    <Card className={styles.card}>
+      <div className={styles.content}>
+        <h2>{"Welcome to the Question Workspace"}</h2>
```

Comment:
I dont think the curly braces are necessary
```suggestion
        <h2>Welcome to the Question Workspace</h2>
```

Reply:
I personally prefer Curly + "" because there is less risk of escaped characters.